sanitiser/_categories.js

```diff
@@ -1,34 +1,53 @@
 
 var check = require('check-types');
+var categoryTaxonomy = require('pelias-categories');
+
+var ERRORS = {
```

Comment:
I like this! Much better than having strings all over the code like we probably do in other sanitisers.